doc/api/v8.md

```diff
@@ -9,6 +9,15 @@ const v8 = require('v8');
 
 *Note*: The APIs and implementation are subject to change at any time.
 
+## v8.cachedDataVersionTag()
+<!-- YAML
+added: REPLACEME
+-->
+
+Returns an integer representing a "version tag" derived from the V8 version,
+command line flags and detected CPU features. This is useful for determining
+whether a [`vm.Script`][] `cachedData` buffer is compatible with another V8.
```

Comment:
I'd s/another V8/this instance of V8/ to emphasize that it applies to the running instance of V8.